pkg/shared/families/rootkits/chkrootkit/chkrootkit.go

```diff
@@ -134,7 +134,7 @@ func (s *Scanner) isValidInputType(sourceType utils.SourceType) bool {
 	switch sourceType {
 	case utils.DIR, utils.ROOTFS:
 		return true
-	case utils.FILE, utils.IMAGE, utils.SBOM:
+	case utils.FILE, utils.IMAGE, utils.SBOM, utils.DOCKERARCHIVE, utils.OCIARCHIVE, utils.OCIDIR:
```

Comment:
we don't want to add the same support also here?

Reply:
missing this one, it should be added here, thanks for catching!